runtime/executor/src/main/java/org/apache/nemo/runtime/executor/bytetransfer/ClosableBlockingQueue.java

```diff
@@ -32,7 +32,7 @@
 public final class ClosableBlockingQueue<T> implements AutoCloseable {
 
   private final Queue<T> queue;
-  private volatile boolean closed = false;
+  private boolean closed = false;
```

Comment:
The original SonarCloud issue was about `throwable`, but not about `closed`.

SonarCloud pointed out that the `volatile` volatilizes the reference only, but not the object.
However, the original purpose of the `volatile` at `throwable` seems to volatilize the reference.

IMHO, these two fields must be volatile. Maybe we can just keep this SonarCloud issue as known-issue.